Formula/xcb-proto.rb

```diff
@@ -30,7 +40,7 @@ def install
       --sysconfdir=#{etc}
       --localstatedir=#{var}
       --disable-silent-rules
-      PYTHON=python3
+      PYTHON=#{which "python3"}
```

Comment:
Are you sure this needs changed? Given this: could `PYTHON=/usr/bin/python3` or similar be used instead?

Reply:
Some linux containers may not have python pre-installed (so people without root will install it somewhere else), or sometimes stuff is not installed to /usr/bin/ on linux. I would rather not hardcode that path and take wathever python 3 is in the path.

Reply:
Fine with me! Does this definitely need changed, though? I'm pretty sure this will already use whatever Python it finds in the PATH?

Reply:
Right, reverted, "python3" alone uses the Python 3 which is in the PATH, so this is what we want.